### PR TITLE
Added config file for GitHub Actions

### PR DESCRIPTION
**Pull request by pchakraborty**
Some of the build settings need to be updated based on the vm settings (e.g. we are building with make -j8), but the build seems to work.

### CODE REVIEW

**Comment by mathomp4** [comment]
@pchakraborty Ooh. Thanks. One question: do you know how expensive this is? I think we only get 2000 minutes per repo per month?

As such, perhaps we should set it to, say, ```on:release:types:published```? I often push a *lot* on Baselibs (at present):
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onevent_nametypes

**Comment by pchakraborty** [comment]
I have updated the trigger for CI build.

**Comment by pchakraborty** [comment]
@mathomp4 Also, this CI build simply builds the baselibs, but does not create (and push) a docker image. That would be the next step.

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: Build baselibs
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build_baselibs:
+    runs-on: ubuntu-latest
```
Comment (mathomp4) [inline .github/workflows/build.yaml]: One more question, @pchakraborty, as I go through my PRs. Why does this run on ubuntu but use EPEL?

**Pull request closed without merging**